test/unit/bandbool.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var assert = require('assert');
+var fixtures = require('../fixtures');
+var sharp = require('../../index');
+
+describe('Bandbool per-channel boolean operations', function() {
+
+  it('\'and\' Operation', function(done) {
+    sharp(fixtures.inputPngBooleanNoAlpha)
+      .bandbool('and')
+      .toBuffer(function(err, data, info) {
+        if (err) throw err;
+        assert.strictEqual(200, info.width);
```

Comment:
Would an assertion on the number of channels in the output image be useful/valid here?
```javascript
assert.strictEquals(1, info.channels)
```